components/datetime/src/neo.rs

```diff
@@ -675,8 +675,8 @@ where
             &<R::Z as ZoneMarkers>::GenericShortV1Marker::bind(provider),
             &<R::Z as ZoneMarkers>::SpecificLongV1Marker::bind(provider),
             &<R::Z as ZoneMarkers>::SpecificShortV1Marker::bind(provider),
-            Some(loader), // fixed decimal formatter
-            Some(loader), // week calculator
+            loader, // fixed decimal formatter
+            loader, // week calculator
```

Comment:
Thought: This regresses on https://github.com/unicode-org/icu4x/issues/4340 (the loaders are supposed to be optional and we just haven't plumbed it in yet), but it's labeled 2.0-stretch. I think it's fine to change this signature for now because the final design might not look like this.